### DISCUSSION TITLE
basis install issue

### DISCUSSION THREAD

**Discussion by chrisgabor007** [seanap/Plex-Audiobook-Guide#86] [General]
Hi, Tried git and manual install but it is not showing up at the agent selection. I looked at the log file and found this line in the server.log

Oct 02, 2022 14:50:38.005 [8224] WARN - Couldn't determine bundle identifier for plug-in at: "C:\Users\Administrator\AppData\Local\Plex Media Server\Plug-ins\Audnexus.bundle"

whats next, thanks chris

**Comment by corbsman** [comment]
Did you restart your plex instance?

**Reply by chrisgabor007** [reply]
yes i did that's the reason for the error in the log,
 i see this board is for a different plug  in. sorry for the cross post.  I just go stuck.

**Comment by seanap** [comment]
May be a permissions issue. 

Which system is your plex server installed on?  

Make sure the folder structure of the plug-in is `\Plex Media Server\Plug-ins\Audnexus.bundle\Contents\...` sometimes when you download git there's an extra Audnexus.bundle folder.

**Reply by chrisgabor007** [reply]
thanks for the quick reply.
it's windows 10 server. the permission is set for the user but I added server\users group with full permission. 
the directory structure is 
...\Audnexus.bundle\.git and other folders 
tried to update 
C:\Users\Administrator\AppData\Local\Plex Media Server\Plug-ins\Audnexus.bundle>git pull
Already up to date.

restarted still the same problem. i might install plex on another instance just to check this plug in, the audiobook agent works fine from webtools 3.0

**Reply by chrisgabor007** [reply]
windows 2019 standard

**Reply by chrisgabor007** [reply]
I did a new plex 64bit Version 1.29.0.6244 install on a clean windows 10 machine. Copied the zip file content to user\AppData\Local\Plex Media Server\Plug-ins folder renamed folder to Audnexus.bundle and started plex server instance. at settings\general\agent\artist no audnexus option. did i miss any steps? same error in server.log, i guess i gave it a go no luck for me.
